cmd/aries-mobile-sdk/.gitignore

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+build/
```

Comment:
Is there a need for `.gitignore`? We have it the root directory.

Reply:
i thought about this and noticed the js-worker folder has a gitignore as well. my thinking was that this could grow to become it's own repo because of the builds and tests. should I  leave that for when the time comes and move remove this?

Reply:
Just comment on your section. We do not need the second file.

```
# mobile-related files 
cmd/aries-mobile-sdk/build/
```